config-schema.json

```diff
@@ -545,6 +560,92 @@
+          ],
+          "properties": {
+            "data": {
+              "$id": "#/properties/integrations/properties/mqtt/properties/data",
+              "type": "array",
+              "title": "The Data Schema",
+              "items": {
+                "$id": "#/properties/integrations/properties/mqtt/properties/data/items",
+                "type": "object",
+                "title": "The Items Schema",
+                "required": [
+                  "data",
+                  "friendly_name",
+                  "id"
+                ],
+                "properties": {
+                  "data": {
+                    "$id": "#/properties/integrations/properties/mqtt/properties/data/items/properties/data",
+                    "type": "object",
+                    "title": "The Data Schema",
+                    "required": [
+                      "ip"
+                    ],
+                    "properties": {
+                      "ip": {
+                        "$id": "#/properties/integrations/properties/mqtt/properties/data/items/properties/data/properties/ip",
+                        "type": "string",
+                        "title": "MQTT Broker IP address",
+                        "examples": [
+                          "192.168.100.1:1883"
+                        ],
+                        "pattern": "^(.*)$"
+                      }
+                    }
+                  },
+                  "friendly_name": {
+                    "$id": "#/properties/integrations/properties/mqtt/properties/data/items/properties/friendly_name",
+                    "type": "string",
+                    "title": "The Friendly_name Schema",
+                    "default": null,
+                    "examples": [
+                      "MQTT"
+                    ],
+                    "pattern": ""
+                  },
+                  "id": {
+                    "$id": "#/properties/integrations/properties/mqtt/properties/data/items/properties/id",
+                    "type": "string",
+                    "title": "The Id Schema",
+                    "default": null,
+                    "examples": [
+                      "mqtt"
+                    ],
+                    "pattern": ""
+                  }
+                }
+              }
+            },
+            "log": {
+              "$id": "#/properties/integrations/properties/mqtt/properties/log",
+              "type": "string",
+              "title": "The Log Schema",
+              "default": null,
+              "examples": [
+                "debug"
+              ],
+              "pattern": ""
+            },
+            "mdns": {
+              "$id": "#/properties/integrations/properties/mqtt/properties/mdns",
+              "type": "string",
+              "title": "The Mdns Schema",
+              "examples": [
+                "_mqtt._tcp"
+              ],
+              "pattern": "^(.*)$"
+            }
+          }
+        },
```

Comment:
MQTT is neither part of remote-software nor this pull request.
Please remove.

Reply:
Ok, this has been removed